backend/compact-connect/lambdas/purchases/tests/function/test_handlers/test_purchase_privileges.py

```diff
@@ -148,6 +159,48 @@ def test_post_purchase_privileges_returns_400_if_selected_jurisdiction_invalid(
 
         self.assertEqual({'message': 'Invalid jurisdiction postal abbreviation'}, response_body)
 
+    @patch('handlers.privileges.PurchaseClient')
+    def test_post_purchase_privileges_returns_400_if_selected_jurisdiction_matches_existing_license(
+        self, mock_purchase_client_constructor
+    ):
+        from handlers.privileges import post_purchase_privileges
+
+        self._when_purchase_client_successfully_processes_request(mock_purchase_client_constructor)
+
+        event = self._when_testing_provider_user_event_with_custom_claims()
+        event['body'] = _generate_test_request_body(['oh'])
+
+        resp = post_purchase_privileges(event, self.mock_context)
+        self.assertEqual(400, resp['statusCode'])
+        response_body = json.loads(resp['body'])
+
+        self.assertEqual(
+            {'message': "Selected privilege jurisdiction 'oh' matches license jurisdiction"}, response_body
```

Comment:
i'm confused -wasn't this changed to kentucky?

Reply:
On line 171 we are passing in 'oh' as the selectedJurisdiction in the post body

Reply:
I've added comments to the test cases to explain the setup